terraform/modules/emr/templates/emr/poll_status_table.sh

```diff
@@ -27,8 +27,17 @@ MESSAGE="TIMEOUT set to: \"$TIMEOUT\""
 echo $MESSAGE
 log_message $MESSAGE "INFO" "NOT_SET" $PROCESS_ID "batch_emr" "poll_status_table.sh" "NOT_SET"
 
+if [[ -n "$3" ]]; then
+  EXPORT_DATE="$3"
+else EXPORT_DATE="current_date()"
```

Comment:
This isnt going to work if $3 isnt passed. It will use the string comparision 'current_date()'.  Can you instead use todays date explicitly. e.g. EXPORT_DATE=$(date +"%Y-%m-%d")